boards/x220/x220.config

```diff
@@ -17,7 +17,9 @@ CONFIG_QRENCODE=y
 CONFIG_TPMTOTP=y
 CONFIG_DROPBEAR=y
 
-CONFIG_LINUX_USB=y
+CONFIG_LINUX_USB_EHCI=y
```

Comment:
Instead of changing every config to specifically bring in the different USB drivers, let them define `CONFIG_LINUX_USB` and then `modules/linux` can check to see if that variable is defined and create the various drivers.  That way only platforms that want to specialize need to tweak their `CONFIG_LINUX_USB_EHCI` etc definitions.

Reply:
ok I will try to do that